superbuild/projects_modules/eigen.cmake

```diff
@@ -75,7 +75,7 @@ ExternalProject_Add_Step(${ep} post_install
 ## Set variable to provide infos about the project
 ## #############################################################################
 
-set(${ep}_ROOT ${build_path} PARENT_SCOPE)
+set(${ep}_ROOT ${EP_PATH_SOURCE}/${ep} PARENT_SCOPE)
```

Comment:
Can you undo this change too, and put back the line '-DEigen3_ROOT:PATH=${eigen_ROOT}' in `music-plugins.cmake`? It is needed for the compilation on github actions.

Reply:
To me more precise:
- `-DEigen3_ROOT:PATH=${eigen_ROOT}` is required for the compilation on github actions to work with the cached external projects.
- `set(${ep}_ROOT ${build_path} PARENT_SCOPE)` should be left as is because the `_ROOT` variables are meant to point to package roots (the build or install directory, not the source).